mapper/census.go

```diff
@@ -257,6 +257,11 @@ func CreateCensusDatasetLandingPage(ctx context.Context, req *http.Request, base
 
 	p.BetaBannerEnabled = true
 
+	p.ShowCensusBranding = false
+	if d.Survey == "census" {
+		p.ShowCensusBranding = true
+	}
+
```

Comment:
This could be a one liner
```suggestion
	p.ShowCensusBranding = d.Survey == "census"

```